catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -52,25 +54,59 @@
 
   private static final String NOT_AVAILABLE = "N/A";
 
+  @Option(
+    name = "--input",
+    required = false,
+    aliases = {"-i", "-input"},
+    multiValued = false,
+    description = "Displays only input transformers"
+  )
+  boolean inputOption = false;
+
+  @Option(
+    name = "--metacard",
+    required = false,
+    aliases = {"-m", "-metacard"},
+    multiValued = false,
+    description = "Displays only metacard transformers"
+  )
+  boolean metacardOption = false;
+
   @Override
   protected Object executeWithSubject() throws Exception {
 
-    // ServiceReferences are needed to get property information from InputTransformers
-    List<ServiceReference<InputTransformer>> serviceReferences =
-        (List<ServiceReference<InputTransformer>>)
-            bundleContext.getServiceReferences(InputTransformer.class, "(id=*)");
+    if (inputOption) {
+      printTransformers(InputTransformer.class, INPUT);
+      return null;
+    }
+    if (metacardOption) {
+      printTransformers(MetacardTransformer.class, METACARD);
```

Comment:
❓ Did we want to pull separate service properties for each class? For instance, the service property `description` for MetacardTransformers?

Reply:
I liked your idea of adding another option that would just print all properties